Sources/PCUserSubscription.swift

```diff
@@ -636,6 +488,7 @@ extension PCUserSubscription {
             return
         }
 
+        // TODO: Why not weak self here?
```

Comment:
Should these `TODO` have a card?

Reply:
Maybe - they're not high priority and more gentle reminders for me (or someone else) to think about when they're in the code next. I just spotted them as a passing thing